### PR TITLE
Repo and Pod Boilerplate

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@justinvdm, @rudigiesler ready for review

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+here = os.path.abspath(os.path.dirname(__file__))
+with open(os.path.join(here, 'README.rst')) as f:
+    readme = f.read()
+
+with open(os.path.join(here, 'VERSION')) as f:
+    version = f.read().strip()
+
+setup(name='casepro.pods.familyconnect',
+      version=version,
+      description=('.'),
```
Comment (justinvdm) [inline setup.py]: Maybe we should add a description here (just noticed the dummy pod also doesn't have a description, will fix, sorry about that).

**Comment by justinvdm** [comment]
One very minor comment, looks good.

File: setup.py (hunk 1 above)
Comment (KaitCrawford) [inline setup.py]: I totally didn't copy the dummy pod stuff ;) :P

**Comment by KaitCrawford** [comment]
@justinvdm ready again

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**